src/MenuItem.react.js

```diff
@@ -33,7 +33,7 @@ const MenuItem = React.createClass({
           'active': active,
           'disabled': disabled,
         }, className)}>
-        <a style={{ cursor: 'pointer' }} onClick={this._handleClick}>
+        <a role="button" onClick={this._handleClick}>
```

Comment:
nit: can you please alphabetize the props? The linter will complain otherwise. Thanks.